src/index.ts

```diff
@@ -134,17 +140,26 @@ export class I2C extends Peripheral {
     super.destroy();
   }
 
-  private getDevice(address: number) {
+  private getDevice(address: number, busNumber?: number) {
     let device = this.devices[address];
 
     if (device === undefined) {
-      device = openSync(getBoardRevision() === VERSION_1_MODEL_B_REV_1 ? 0 : 1);
+      if (busNumber === undefined) {
+        busNumber = this.busNumber;
+      }
+      device = openSync(busNumber);
       this.devices[address] = device;
     }
 
     return device;
   }
 
+  public configure(options?: any): void {
```

Comment:
Ahhh, interesting, I did not know that. OK then, so the way forward will be to handle this "figure out what bus a given address is on" will be in raspi-io-core. Switching busses will happen there, not here.

This is curious though, with the APIs defined the way they are for IO plugins, it's not possible to talk to two devices with the same address but each on a different bus. While unfortunate, it _does_ make things easier because we can have a lookup map in raspi-io-core that maps from address to bus/raspi-i2c instance.

For this PR, we still need to make the changes I requested.